lib/doorkeeper/oauth/authorization/uri_builder.rb

```diff
@@ -20,7 +20,7 @@ def uri_with_fragment(url, parameters = {})
         end
 
         def build_query(parameters = {})
-          parameters = parameters.reject { |k, v| v.blank? }
+          parameters = parameters.reject { |_k, v| v.blank? }
```

Comment:
Ok, and what about https://github.com/doorkeeper-gem/doorkeeper/blob/master/lib/doorkeeper/config.rb#L149 ? No args is used in the `define_method` block.